demos/hr_agent/main.py

```diff
@@ -0,0 +1,103 @@
+import os
+import json
+import pandas as pd
+from fastapi import FastAPI, HTTPException
+from pydantic import BaseModel, Field
+from typing import Optional
+from enum import Enum
+from slack_sdk import WebClient
+from slack_sdk.errors import SlackApiError
+
+app = FastAPI()
+workforce_data_df = None
+
+with open("workforce_data.json") as file:
+    workforce_data = json.load(file)
+    workforce_data_df = pd.json_normalize(
+        workforce_data, record_path=["regions"], meta=["point_in_time", "satisfaction"]
+    )
+
+
+# Define the request model
+class WorkforceRequset(BaseModel):
+    region: str
+    staffing_type: str
+    point_in_time: Optional[int] = None
+
+
+class SlackRequest(BaseModel):
+    slack_message: str
+
+
+class WorkforceResponse(BaseModel):
+    region: str
+    staffing_type: str
+    headcount: int
+    satisfaction: float
```

Comment:
looks like this is not passed in from arch_config, as I didn't see it in parameters list for the prompt_target `workforce`

Reply:
this is for response, not request